controller/users_blackbox_test.go

```diff
@@ -822,12 +869,13 @@ func (s *TestUsersSuite) TestListUsersByEmailNotModifiedUsingIfNoneMatchHeader()
 
 func (s *TestUsersSuite) createRandomUser(fullname string) account.User {
```

Comment:
Maybe we can replace this function by test.CreateTestUser() too?

Reply:
For consistency's sake, I will keep it as it is - because if change here, it *should* be changed in all other places in this file. 
In fact in this PR, https://github.com/fabric8-services/fabric8-auth/pull/227/files#diff-cc740fdbd6f22db6da588050e1b7d5a4R78
I have included a util method there too :)

Reply:
So? Is this a huge refactoring? :) I mean we should try to keep our tests clean. And try to improve our tests when it makes sense as well as we improve our main code. And if you already changing these tests it's fine to do some moderate test refactoring of the related code here. But it's also OK to do the test refactoring in the separate PR but I don't want us to create report more technical debt issues and forget about them ;)

Reply:
Cool, I will make refactor in this PR :)

Reply:
done - https://github.com/fabric8-services/fabric8-auth/pull/237/commits/f826965bb991df019f209ecb399ab385edbfab4e